File: Electronics_Shop_17.Model/Requests/UpdateAccessoryProperty.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Model.Requests
{
    public class UpdateAccessoryProperty
    {
        public string? PropertyName { get; set; }
        public string? PropertyValue { get; set; }
        public int? AccessoryId { get; set; }
    }
}

File: angular17_web/src/app/models/accessory-property/update.accessory.property.ts
export class UpdateAccessoryProperty {
  propertyName?: string;
  propertyValue?: string;
  accessoryId?: number;

  constructor(data?: any) {
    if (!data) return;

    this.propertyName = data.propertyName;
    this.propertyValue = data.propertyValue;
    this.accessoryId = data.accessoryId;
  }
}
